quickwit-cluster/src/client.rs

```diff
@@ -0,0 +1,70 @@
+//  Quickwit
+//  Copyright (C) 2021 Quickwit Inc.
+//
+//  Quickwit is offered under the AGPL v3.0 and as commercial software.
+//  For commercial licensing, contact us at hello@quickwit.io.
+//
+//  AGPL:
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Affero General Public License as
+//  published by the Free Software Foundation, either version 3 of the
+//  License, or (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Affero General Public License for more details.
+//
+//  You should have received a copy of the GNU Affero General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::net::SocketAddr;
+
+use http::Uri;
+use tonic::transport::Channel;
+use tonic::transport::Endpoint;
+
+use quickwit_proto::cluster_service_client::ClusterServiceClient;
+use quickwit_proto::search_service_client::SearchServiceClient;
+
+/// Create a ClusterServiceClient with SocketAddr as an argument.
+/// It will try to reconnect to the node automatically.
+pub async fn create_cluster_service_client(
```

Comment:
will it ever be used? I think this service is only here for debugging & admin purpose isn't it?
If so I think we can remove this function.

Reply:
This service is only used for debugging and administration, so I don't need the client creation function. I'll emove it.

Reply:
Changed.
Please see: https://github.com/quickwit-inc/quickwit/pull/143/commits/cc6acc406a9469092ca23cf4d7bea59acb481db4